src/Analyzer.Core/TemplateAnalyzer.cs

```diff
@@ -101,12 +101,60 @@ public IEnumerable<IEvaluation> AnalyzeTemplate(string template, string paramete
             }
         }
 
-        private static string LoadRules()
+        /// <summary>
+        /// Loads rules.
+        /// </summary>
+        /// <returns>Loaded rules.</returns>
+        public static string LoadRules()
         {
             return File.ReadAllText(
                 Path.Combine(
                     Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location),
                     "Rules/BuiltInRules.json"));
         }
+
+        /// <summary>
+        /// Loads a configurations file. If no file was passed, checks the default directory for this file.
+        /// </summary>
+        /// <param name="configurationsFilePath">The path to a configuration file to read.</param>
+        /// <returns>Configuration file path contents if a file exists.</returns>
+        private string GetConfigurationFileContents(FileInfo configurationsFilePath)
+        {
+            try
+            {
+                string path;
+                if (configurationsFilePath != null)
+                {
+                    path = configurationsFilePath.FullName;
+                }
+                else
+                {
+                    path = Path.Combine(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location),
```

Comment:
Missing tests for when 
1. `configurationFilePath` is null
2. Default configuration file is not present

Reply:
1. I tested it in JsonRuleEngineTests.FilterRules_ValidInputValues_ReturnCorrectFilteredRules, but makes sense to have it here too as it is one step prior. I added in a test for it. The way that makes more sense to me to test this is if the # of rules does not change, but that would involve changing a lot to make those values accessible to this class.
2. It is implied by (1). 
The test I do not have is if it is in the default location, but then the no configuration FilterRules test would not work (as there is a configuration) and the path with it being a test would get messy.